File: Codeforces/ECNA 2023/d.py
arr = [int(i) for i in input().split()]
cnt = 0
for i in range(0, 10, 2):
    cnt += arr[i]*arr[i+1]
n, kwf = [int(i) for i in input().split()]
cnt = cnt//5

print(int((n*cnt)//kwf))

File: Codeforces/ECNA 2023/d.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long int ll;

int main(){
    vector<int> arr;
    for (int i = 0; i < 10; i++) {
        int x; cin >> x;
        arr.push_back(x);
    }

    int n, kwf; cin >> n >> kwf;
    int cnt = 0;
    for (int i = 0; i < 5; i++) {
        cnt += arr[i]*arr[i+5];
    }

    cnt /= 5;
    cout << (cnt*n)/kwf << endl;
}